config/config.go

```diff
@@ -173,18 +173,22 @@ func NewConfig(path string) (*Config, error) {
 	c := newConfig()
 	c.ConfigFile = path
 
+	log.Debugf("Loading choria configuration from '%s'", path)
 	err := parseConfig(path, c, "", c.rawOpts)
 	if err != nil {
 		return nil, err
 	}
 
-	err = parseConfig(c.ConfigFile, c.Choria, "", c.rawOpts)
-	if err != nil {
-		return nil, err
-	}
-
```

Comment:
The exiting code seems to load the configuration twice.  I remove this second call to `parseConfig()` assuming that was safe, but maybe I am missing something…

Reply:
Ahhhh… `c` != `c.Choria` (2nd parameter)… This is breaking the test suite